telegramer/core.py

```diff
@@ -236,7 +238,11 @@ def enable(self):
                         log.debug(prelog() + 'Notify: ' + str(self.notifylist))
 
                 reactor.callLater(2, self.connect_events)
-
+                try :
+                    self.checkSpeed_timer = LoopingCall(self.checkSpeed)
+                    self.checkSpeed_timer.start(int(self.config['user_timer']), now=False)
```

Comment:
Maybe add: 
` if self.config['minimum_speed'] > -1:`
here so we wan't start the timer if no speed limit is given instead of checking it each time/ every minute